scripts/ldr

```diff
@@ -0,0 +1,177 @@
+    roachprod create $B \
+      --clouds gce --gce-machine-type n2-standard-16 --nodes 3 --username "$USER" --lifetime 24h "$@"
+    roachprod stage $A cockroach
+    roachprod stage $A cockroach
+    roachprod stage $B workload
+    roachprod stage $B workload
+    ;;
+
+  "jobs")
+    shift
+    case "${1:-}" in
+    "start")
+      roachprod sql $A:1 -- -e "SET CLUSTER SETTING kv.rangefeed.enabled = true"
+      roachprod sql $B:1 -- -e "SET CLUSTER SETTING kv.rangefeed.enabled = true"
+      roachprod sql $A:1 -- -e "USE ycsb; SELECT crdb_internal.start_logical_replication_job($(roachprod pgurl --insecure $B:1), ARRAY['usertable']);"
+      roachprod sql $B:1 -- -e "USE ycsb; SELECT crdb_internal.start_logical_replication_job($(roachprod pgurl --insecure $A:1), ARRAY['usertable']);"
+      ;;
+    "pause")
+      roachprod sql $A:1 -- -e "PAUSE JOBS (WITH x AS (SHOW JOBS) SELECT job_id FROM x WHERE job_type = 'LOGICAL REPLICATION' AND status = 'running');"
+      roachprod sql $B:1 -- -e "PAUSE JOBS (WITH x AS (SHOW JOBS) SELECT job_id FROM x WHERE job_type = 'LOGICAL REPLICATION' AND status = 'running');"
+      ;;
+    "resume")
+      roachprod sql $A:1 -- -e "RESUME JOBS (WITH x AS (SHOW JOBS) SELECT job_id FROM x WHERE job_type = 'LOGICAL REPLICATION' AND status = 'paused');"
+      roachprod sql $B:1 -- -e "RESUME JOBS (WITH x AS (SHOW JOBS) SELECT job_id FROM x WHERE job_type = 'LOGICAL REPLICATION' AND status = 'paused');"
+      ;;
+    "cancel")
+      roachprod sql $A:1 -- -e "CANCEL JOBS (WITH x AS (SHOW JOBS) SELECT job_id FROM x WHERE job_type = 'LOGICAL REPLICATION' AND status = 'running' OR status = 'paused');"
+      roachprod sql $B:1 -- -e "CANCEL JOBS (WITH x AS (SHOW JOBS) SELECT job_id FROM x WHERE job_type = 'LOGICAL REPLICATION' AND status = 'running' OR status = 'paused');"
+      ;;
+    "stats")
+      roachprod sql $A -- -e "select * from crdb_internal.cluster_replication_node_streams;"  --format=table
+      roachprod sql $B -- -e "select * from crdb_internal.cluster_replication_node_streams;"  --format=table
+      ;;
+    *)
+      echo "unknown command '$1'; useage: $0 {start|pause|resume|cancel|stats}"
+      exit 1
+      ;;
+    esac
+    ;;
+
+  "workload")
+    shift
+    case "${1:-}" in
+      "init")
+        roachprod run $A:1 "./workload init ycsb --drop --families=false --splits 1000 --seed 1665361151361545632 $(roachprod pgurl --insecure $A)"
+        roachprod run $B:1 "./workload init ycsb --drop --families=false --splits 1000 --seed 665361151356328 $(roachprod pgurl --insecure $B)"
+        roachprod sql $A:1 -- -e "ALTER TABLE ycsb.public.usertable ADD COLUMN crdb_internal_origin_timestamp DECIMAL NOT VISIBLE DEFAULT NULL ON UPDATE NULL"
+        roachprod sql $B:1 -- -e "ALTER TABLE ycsb.public.usertable ADD COLUMN crdb_internal_origin_timestamp DECIMAL NOT VISIBLE DEFAULT NULL ON UPDATE NULL"
```

Comment:
In my latest copy, I also have:

`"ALTER TABLE ycsb.public.usertable SET (ttl_expire_after = '3 hours');"`

on both sides.  If we don't do this, we'll need infinite disk space for a long run.